helm/kubernetes-cluster-prep/README.md

```diff
@@ -0,0 +1,288 @@
+# Helm Chart for Conjur Kubernetes Cluster Preparation
+
+## Table of Contents
+
+- [Helm Chart for Conjur Kubernetes Cluster Preparation](#helm-chart-for-conjur-kubernetes-cluster-preparation)
+  * [Table of Contents](#table-of-contents)
+  * [Overview](#overview)
+    + [Objects Created](#objects-created)
+  * [Preparing the Kubernetes Cluster for Conjur Authentication](#preparing-the-kubernetes-cluster-for-conjur-authentication)
+  * [Examples: Running Helm Install](#examples--running-helm-install)
```

Comment:
I think the double dashes `--` need to be replace with a single dash `-` in this line (line 10) and the next line. Otherwise, the link doesn't work.

Reply:
updated